README.md

```diff
@@ -36,7 +36,7 @@ Break this file up into two separate files:
 Here's `app/assets/javascripts/i18n/translations.js.i18njs`:
 
 ```yaml
----
+{}
```

Comment:
I think changing the docs makes sense. I thought the empty `---` yaml front matter was something I used in the tests, but it looks like that's not the case.